rootston/cursor.c

```diff
@@ -309,24 +309,9 @@ static void notify_relative_motion(struct roots_seat *seat, uint64_t time_msec,
 		double dx, double dy, double dx_unaccel, double dy_unaccel) {
 	struct wlr_relative_pointer_manager_v1 *relative_pointer_manager =
 		seat->input->server->desktop->relative_pointer_manager;
-
```

Comment:
what is the point of having this function at all, it is just a one-line wrapper for `wlr_relative_pointer_manager_v1_send_relative_motion` ...

Reply:
Good catch, fixed